Rechtspraak.nl.js

```diff
@@ -0,0 +1,259 @@
+{
+	"translatorID": "aede3fda-1894-4dfc-8bca-1c3463f11076",
+	"label": "Rechtspraak.nl",
+	"creator": "Pieter van der Wees",
+	"target": "^https?://*uitspraken\\.rechtspraak\\.nl/",
```

Comment:
I don't think this regex is right. The `*` causes it to match zero or more of the previous character, i.e., zero or more slashes.

Will the URL ever be anything other than `uitspraken.rechtspraak.nl` or `deeplink.uitspraken.rechtspraak.nl`? If not, you can just do

```suggestion
	"target": "^https?://(deeplink\\.)?uitspraken\\.rechtspraak\\.nl/",
```

Reply:
deeplink.rechtspraak.nl just redirects to uitspraken.rechtspraak.nl, so we do not need to add that to the regex. I have simply removed the `*`.